scrapscript.py

```diff
@@ -1320,6 +1323,96 @@ def deserialize(msg: str) -> Object:
     return Object.deserialize(decoded)
 
 
+class ScrapMonad:
```

Comment:
Still have a feeling that monad and server stuff (minus the CLI for it) should be a separate file since it's not part of the core interpreter, but up to you

Reply:
I will leave it for now, I think

Reply:
The Python bit is very small